src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  useParams
+} from "react-router-dom";
+import './App.css';
+import ListPage from './ListPage';
+import CreatePage from './CreatePage';
+import DetailPage from './DetailPage';
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <div className="header-div">
+        <h2>The Worst Page in the History of Pages</h2>
+        <p>
+          <Link to="/">
+            Home
+          </Link>
+        </p>
+        <p>
+          <Link to="/create">
+            Add a Character
+          </Link>
+          </p></div>
+        
+        <Switch>
+          <Route path="/characters/:id" exact render={(routerProps) => <DetailPage {...routerProps} />}
```

Comment:
I'd imagine part of the reason Netlify isn't deploying, is you haven't created a route for the ```/``` route.   You just have a link to it in the header, but no way for React to load a page.